test/plugin/test_out_detect_exceptions.rb

```diff
@@ -98,6 +114,80 @@ def test_exception_detection
     assert_equal(make_logs(t, *messages), d.events)
   end
 
+  def test_ignore_exception_nested_in_json
```

Comment:
I agree that the two test scenarios are very redundant, do you think, json or not, we should just choose one of the two? After all is said and done, they're testing the same thing, but is one more clear than the other to choose as the primary test case? Which one of the two test cases do you prefer?